### ISSUE TITLE
JAWS not reading the infotab titles while loading new tabs in Edge and Chrome

### ISSUE THREAD

**Issue by HarshAnand19** [FreedomScientific/VFO-standards-support#922]
Summary :
While using JAWS 2025 Screen Reader with infotabs, the Screen Reader is not able to read the tabtitles of new tabs that are being fetched while navigating towads right. JAWS 2025 just reads the loading message while fetching the data of new infotab and then does not read the tab titles.This issue is present on Edge and Chrome browser while with firefox it is reading fine.
With NVDA it is reading properly across all the browsers.

Tools Used -
JAWS 2025.2506.170
Browsers - Edge and Chrome
OS - Windows 11

Infotabs -

<img width="1380" height="248" alt="Image" src="https://github.com/user-attachments/assets/98f4ed2f-9dfc-4b17-87ca-f1b0280cfa68" />

HTML Attribute - 

<img width="1872" height="525" alt="Image" src="https://github.com/user-attachments/assets/ffc1530d-1f1f-4b46-ab0f-367fafd532f1" />

**Comment by JeremiahRogersVispero** [comment]
@HarshAnand19 , can you please provide code or a page where the issue occurs, along with specific steps to reproduce?
